### PR TITLE
Change HEATFLUX to TOTAL_HEATFLUX in output files so that optimizatio…

### PR DESCRIPTION
**Pull request by MiGaff**
Change HEATFLUX to TOTAL_HEATFLUX in output files so that optimization works correctly with  OPT_OBJECTIVE= TOTAL_HEATFLUX
 
Additionally, a small change to updateHistoryMap.py has been made (a "/" in a path was missing), and historyMap.py has been modified by running updateHistoryMap.py

This fixes [#946](https://github.com/su2code/SU2/issues/946)

## PR Checklist
*Put an X by all that apply. You can fill this out after submitting the PR. If you have any questions, don't hesitate to ask! We want to help. These are a guide for you to know what the reviewers will be looking for in your contribution.*

- [x] I am submitting my contribution to the develop branch.

- [ ] My contribution generates no new compiler warnings (try with the '-Wall -Wextra -Wno-unused-parameter -Wno-empty-body' compiler flags).
- [ ] My contribution is commented and consistent with SU2 style.
- [ ] I have added a test case that demonstrates my contribution, if necessary.
- [ ] I have updated appropriate documentation (Tutorials, Docs Page, config_template.cpp) , if necessary.

### CODE REVIEW

File: SU2_CFD/src/output/CFlowCompOutput.cpp [hunk 1]
```diff
@@ -231,7 +231,7 @@ void CFlowCompOutput::SetHistoryOutputFields(CConfig *config){
 
   ///   /// BEGIN_GROUP: HEAT_COEFF, DESCRIPTION: Heat coefficients on all surfaces set with MARKER_MONITORING.
   /// DESCRIPTION: Total heatflux
-  AddHistoryOutput("HEATFLUX", "HF",      ScreenOutputFormat::SCIENTIFIC, "HEAT", "Total heatflux on all surfaces set with MARKER_MONITORING.", HistoryFieldType::COEFFICIENT);
+  AddHistoryOutput("TOTAL_HEATFLUX", "HF",      ScreenOutputFormat::SCIENTIFIC, "HEAT", "Total heatflux on all surfaces set with MARKER_MONITORING.", HistoryFieldType::COEFFICIENT);
```
Comment (pcarruscag) [inline SU2_CFD/src/output/CFlowCompOutput.cpp:234]: Is this change required for the fix? (thinking about backwards compatibility, documentation, etc.)

Comment (MiGaff) [reply SU2_CFD/src/output/CFlowCompOutput.cpp:234]: I think so. According to the AddHistoryOutput function, the first parameter is:

> name - Name for referencing it (in the config file and in the code).

If I understand correctly this parameter is called TOTAL_HEATFLUX everywhere in the code, but not in these 2 files.

File: SU2_PY/updateHistoryMap.py [hunk 2]
```diff
@@ -95,7 +95,7 @@ def parse_output(files):
 
 
     outputFields.update(addedOutputFields)
-    f = open(os.path.join(su2_home) + 'SU2_PY/SU2/io/historyMap.py', 'w')
+    f = open(os.path.join(su2_home) + '/SU2_PY/SU2/io/historyMap.py', 'w')
```
[the remark this answers is not in the document]
Comment (MiGaff) [reply SU2_PY/updateHistoryMap.py:98]: Yes, that is definitely more elegant.

File: SU2_CFD/src/output/CFlowCompOutput.cpp (hunk 1 above)
Comment (pcarruscag) [reply SU2_CFD/src/output/CFlowCompOutput.cpp:234]: CHT people @oleburghardt @TobiKattmann, can you give an opinion here, how does this influence existing tutorials, testcases, etc.?

Comment (oleburghardt) [reply SU2_CFD/src/output/CFlowCompOutput.cpp:234]: Hi, these changes shouldn't affect them as everything is done internally there (the value always comes from `solver->GetTotal_HeatFlux()`, if needed).

So from my side I'd be fine with changing the name. For tutorials and verification it's just about producing and checking the gradient, I've never checked for compatibility with the python framework.
Would be great if it can be integrated into the regression tests some day.. O:-)
